mkdocs/utils.py

```diff
@@ -216,12 +214,10 @@ def create_relative_media_url(nav, url):
     return relative_url
 
 
-def normalise_path(path):
-    """
-    Normalise POSIX and NT paths to be consistently POSIX style.
-    """
+def path_to_url(path):
+    """Convert a system path to a URL."""
 
-    if ntpath.sep in path:
-        path = path.replace(ntpath.sep, posixpath.sep)
+    if os.path.sep == '/':
+        return path
 
-    return path
+    return pathname2url(path)
```

Comment:
Are you sure pathname2url does what you want? In my experience it url encodes the path but does not flip backward slashes to forward slashes. It may make sense to call pathname2url on all paths converted to urls (system files may contain spaces or other characters generally not allowed in a URL), but only after the slashes are normalized.